File: 백준/Bronze/2480. 주사위 세개/주사위 세개.py
#2480

a, b, c = input().split()
a = int(a)
b = int(b)
c = int(c)

if 1<=a<=6 and 1<=b<=6 and 1<=c<=6:
    if a==b and b==c:
        print(10000+a*1000)
    elif (a==b and b!=c) or (a==c and c!=b):
        print(1000+a*100)
    elif (b==c and b!=a):
        print(1000+b*100)
    elif a!=b and a!=c and b!=c:
        print(max([a,b,c])*100)
else:
    print("조건을 만족하지 않습니다.")

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		
		StringTokenizer st = new StringTokenizer(br.readLine());
		int a = Integer.parseInt(st.nextToken());
		int b = Integer.parseInt(st.nextToken());
		int c = Integer.parseInt(st.nextToken());
		
		if(a==b&&b==c) System.out.println(10000+a*1000);
		else if((a==b&&b!=c) || (a==c&&a!=b)) System.out.println(1000+a*100);
		else if((b==c&&a!=b)) System.out.println(1000+b*100);
		else if(a!=b&&b!=c&&c!=a) System.out.println(Math.max(a, Math.max(b, c))*100);
	}
}
